include/wx/apptrait.h

```diff
@@ -209,13 +211,16 @@ class WXDLLIMPEXP_BASE wxConsoleAppTraitsBase : public wxAppTraits
     virtual bool HasStderr() wxOVERRIDE;
 
     // the GetToolkitVersion for console application is always the same
-    virtual wxPortId GetToolkitVersion(int *verMaj = NULL, int *verMin = NULL) const wxOVERRIDE
+    wxPortId GetToolkitVersion(int *verMaj = NULL,
```

Comment:
Initially I didn't make these changes but then re-read the [guidelines](http://www.wxwidgets.org/develop/coding-guidelines/#use_override):

>Notice that the virtual keyword is redundant as wxOVERRIDE already identifies the method as being >virtual, so it can be omitted, even in most of the existing code both keywords are used together as >wxOVERRIDE is a relatively recent addition and was just added to the existing method declarations.

(which you are probably well familiar with...) and interpreted that as it would eventually be better if the redundancy is removed and that it would be all right to do it for those lines that are already being changed anyway. It's of course still unrelated though and I will change it back again. Another time it might be better to do it in one go if the guideline still holds up.